src/utils/apiClient.ts

```diff
@@ -0,0 +1,13 @@
+import axios, { AxiosInstance } from 'axios';
+
+const createApiClient = (baseURL: string, apiKey: string): AxiosInstance => {
+  return axios.create({
+    baseURL,
+    headers: {
+      ApiKey: apiKey,
```

Comment:
this is great!!
you can also put the api key right in here since its going to be the same for all apis